packages/plugins/Filter/README.md

```diff
@@ -0,0 +1,117 @@
+# Filter
+
+## Scope
+
+The Filter plugin can be used to filter arbitrary configurable vector layers by their properties.
+
+### Configuration
+
+#### filter
+
+| fieldName | type | description |
+| - | - | - |
+| layers | Record<string, filterConfiguration> | Maps layer id to filter configuration. |
+
+```
+The following chapters contain drafts in this format. Please mind that they neither represent UI nor localisation, but are merely there to communicate the idea with an example.
+```
+
+#### filter.filterConfiguration
+
+| fieldName | type | description |
+| - | - | - |
+| categories | category[]? | Category filter definition to filter features by their property values. |
+| time | time? | Time filter definition so filter features by a time property. |
+
+##### filter.filterConfiguration.category
+
+| fieldName | type | description |
+| - | - | - |
+| targetProperty | string | Target property to filter by. This is the name (that is, key) of a feature property. |
+| pattern | string? | Pattern the target string uses for its date formatting. Defaults to `'YYYY-MM-DD'`. Only 'Y', 'M', and 'D' are interpreted. All other characters are considered filler. |
```

Comment:
The pattern is currently for a date, since only 'days' is supported at the time being. However, this may change in later iterations.

An example has been added.

```
/**
 * Feature holds date property as e.g. "20143012", where 2014 is the
 * year, 30 the day, and 12 the month.
 */
pattern: 'YYYYDDMM'
```